doc/spec_TransferElement.dox

```diff
@@ -103,7 +103,8 @@ This document describes the behaviour of the TransferElement base class, the NDR
     - ChimeraTK::Void
   - 2.2 Applications select a data type for the buffer via the UserType template argument of Device::getZzzRegisterAccessor() resp. DeviceBackend::getRegisterAccessor().
   - 2.3 If needed, the register value is converted between the actual register's data type and the application-selected UserType.
-    - 2.3.1 When the actual type is Void, the conversion is 0 or false in the respective UserType.
+    - 2.3.1 When the UserType is ChimeraTK::Void, reading from the device discards any actual data. The resulting user buffer is of type <vectorctk::Void>.
+    - 2.3.2 Converting into another UserType from ChimeraTK::Void will fill the user buffer with 0, false or the respective value for that type.
```

Comment:
" into another UserType from ChimeraTK::Void"
That's wrong. The user type can either be ctk::Void or something else, but never both in the same buffer, so you cannot convert into it.
This point should be: On the device the "register" is conceptually void (e.g. an interrupt without data, or a function without argument that triggers an action), but the user requests a UserType different than void.
Again missing: What is the shape of the buffer? writing direction